src/conponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  selectedFilter: Status;
+  setSelectedFilter: (filterBy: Status) => void;
+  deleteAllCompleted: () => void;
+  activeTodos: Todo[];
+  completedTodos: Todo[];
```

Comment:
don't pass more information, than you need - this reduces the possibility of making an error

```suggestion
  hasActiveTodosCount: number;
  hasCompletedTodos: boolean;
```